aws_lambda_powertools/utilities/parser/models/ses.py

```diff
@@ -21,7 +20,7 @@ class SesReceiptAction(BaseModel):
 class SesReceipt(BaseModel):
     timestamp: datetime
     processingTimeMillis: PositiveInt
-    recipients: List[EmailStr]
+    recipients: List[str]
```

Comment:
Hey @rubenfonseca @heitorlessa  , that's a breaking change. users are expecting here proper email address validation.
I think a better solution would be:
1. During import try to import EmailStr, if there's an except redefine the type as str
2. add docs and specify that for Ses model, you should install the extra dependency

WDYT?

Reply:
Hi @ran-isenberg , we hear you and thank you for your honest feedback. We don't believe this change will break any customer code, since the field is controlled by AWS. However, out of an abundance of caution, we've issued this PR to v2, so customers have the opportunity to test it with their code.

As we've reiterated before, we don't feel the added benefit of validating something that is controlled by AWS outweighs the size of the library needed for such validation. However we will always be attentive to customer feedback about this or other features